### PR TITLE
Update Quick View button to say "Learn more" until V2

### PR DESCRIPTION
**Pull request by TurnerRiley**
Update Quick View button to say "Learn more" until V2 (currently says "Quick view").

### Original:
quick_view

### New:
learn_more

## Links
Jira ticket: [here](https://codedotorg.atlassian.net/browse/ACQ-587?atlOrigin=eyJpIjoiZjVkNDhlMTM5MmZiNDNjYzlkZjYxYWYzOWZhMThhZTYiLCJwIjoiaiJ9)

## Testing story
Local testing and updating unit tests.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by davidsbailey — APPROVED** [review]

File: apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ const CurriculumCatalogCard = ({
     quickViewButtonDescription={i18n.quickViewDescription({
```
Comment (megcrenshaw) [inline apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx:49]: @dmcavoy Is the screenreader description of the button still okay? It reads "View details about {course_name}"

Comment (dmcavoy) [reply apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx:49]: I think that still makes sense as they will view the details of the course on another page. (Not sure how screenreader friendly that page they will open is tho)

Comment (megcrenshaw) [reply apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx:49]: Yep, that is my thinking too!

Part of what feels hard about the accessibility work is that I feel like we're inadvertently leading teachers and students down paths on our site, only to find that the really "good" stuff (in which learning is actively happening) is not accessible. Not sure what to do about that.

Comment (dmcavoy) [reply apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx:49]: Agreed that is hard. Sam is working on some key student flows over the next several months and we can keep making sure the things we touch meet accessibility guidelines. Over time we will get to a better place. Please continue to be the voice for this stuff. I've learned a lot and its so important.

Comment (megcrenshaw) [reply apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx:49]: Yes, excellent. Very grateful for everyone actively working towards making this better –– thank you, Dani (and others!)

**Pull request merged**